plugins/source/azure/resources/services/sql/database_vulnerability_assesments.go

```diff
@@ -0,0 +1,52 @@
+// Code generated by codegen; DO NOT EDIT.
+package sql
+
+import (
+	"context"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/sql/armsql"
+	"github.com/cloudquery/cloudquery/plugins/source/azure/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/transformers"
+)
+
+func databaseVulnerabillityAssesments() *schema.Table {
+	return &schema.Table{
+		Name:      "azure_sql_database_vulnerability_assesments",
+		Resolver:  fetchDatabaseVulnerabillityAssesments,
+		Multiplex: client.SubscriptionMultiplexRegisteredNamespace("azure_sql_database_vulnerability_assesments", client.Namespacemicrosoft_sql),
+		Transform: transformers.TransformWithStruct(&armsql.DatabaseVulnerabilityAssessment{}),
+		Columns: []schema.Column{
+			{
+				Name:     "id",
+				Type:     schema.TypeString,
+				Resolver: schema.PathResolver("ID"),
+				CreationOptions: schema.ColumnCreationOptions{
+					PrimaryKey: true,
+				},
+			},
+		},
+	}
+}
+
+func fetchDatabaseVulnerabillityAssesments(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- any) error {
+	pp := parent.Item.(*armsql.Server)
```

Comment:
Should this be the parent of the parent?
Also should this be a relation of a databases table? I couldn't find that table anywhere